programs/compressed-token/src/process_transfer.rs

```diff
@@ -33,16 +34,21 @@ pub fn process_transfer<'a, 'b, 'c, 'info: 'b + 'c>(
             &ctx.accounts.authority.key(),
         )?;
 
+    // TODO: check if this is already implemented
```

Comment:
this was a remnant from rebasing spl-compression onto the latest main: the delegation check is already implemented via `get_input_compressed_accounts_with_merkle_context_and_check_signer` correct? @ananas-block

Reply:
Yes delegate checks are implemented but untested.
Also we don't have an instruction to create a compressed token account with a delegate yet.